app/lib/actions/pulp3/repository/delete_orphan_repository_versions.rb

```diff
@@ -0,0 +1,16 @@
+module Actions
+  module Pulp3
+    module Repository
+      class DeleteOrphanRepositoryVersions < Pulp3::Abstract
+        def plan(smart_proxy)
+          plan_self(:smart_proxy_id => smart_proxy.id)
+        end
+
+        def run
+          smart_proxy = SmartProxy.find(input[:smart_proxy_id])
+          ::Katello::Pulp3::Repository.delete_orphan_repository_versions(smart_proxy)
```

Comment:
we probably want this to be an 'abstract_async_task' and have this delete_orphan_repository_versions() method return a list of tasks to track.'